### PR TITLE
Hydro Cannon Fix

### LINKED ISSUE #6695: Hydrocannon button doesn't work
## Testmerges:

Don't think its caused by one

## Reproduction:

1. Click the hydrocannon button
2. Glitch will happen after a couple of tries. The chat will say both hydrocannon on and off and the flamer will still be flamin'

### PR DESCRIPTION
**Pull request by InterroLouis**
Fixes the Hydro Cannon

## About The Pull Request

Fixes the hydro cannon hud button. Now you can actually activate the hydro cannon by clicking the hud button. It also removes ATTACH_ACTIVATION from the hydro cannon, so the load from attachment verb will no longer toggle the hydro cannon. Unique action and the hud button will now be the only ways to toggle it, but I'm assuming people defaulted to unique action anyways. 
Fixes #6695 

## Why It's Good For The Game
It just works. That's really it, the hud button didn't work before and now it does. 
## Changelog
:cl:
fix: Fixed Hydro Cannon hud button.
refactor: Hydro Cannon only works via unique action and hud button, not load from attachment.
/:cl:

### CODE REVIEW

**Comment by lbnesquik** [comment]
Based

File: code/datums/actions/item_action.dm [hunk 1]
```diff
@@ -117,7 +117,6 @@
 	RegisterSignal(holder_flamer, COMSIG_ITEM_HYDRO_CANNON_TOGGLED, .proc/update_toggle_button_icon)
 
 /datum/action/item_action/toggle_hydro/action_activate()
-	. = ..()
 	holder_flamer.unique_action(owner)
```
Comment (TiviPlus) [inline code/datums/actions/item_action.dm:120]: Explain this

Comment (InterroLouis) [reply code/datums/actions/item_action.dm:120]: It was calling the above proc which ended up making the action happen twice.

Comment (TiviPlus) [reply code/datums/actions/item_action.dm:120]: What above proc
Why didnt you just delete the unique action call

Comment (InterroLouis) [reply code/datums/actions/item_action.dm:120]: I tried that and while it worked for making it not double, it didn't update the hud icon.

Comment (TiviPlus) [reply code/datums/actions/item_action.dm:120]: This proc should be calling parent look into why its being called twice instead

Comment (SplinterGP) [reply code/datums/actions/item_action.dm:120]: it uses ui_action_click()

File: code/datums/actions/item_action.dm [hunk 2]
```diff
@@ -118,7 +118,6 @@
 
 /datum/action/item_action/toggle_hydro/action_activate()
 	. = ..()
```
Comment (TiviPlus) [inline code/datums/actions/item_action.dm:120]:
```suggestion
```

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**